jerry-main/main-utils.c

```diff
@@ -148,6 +148,38 @@ test262_eval_script (const jerry_value_t func_obj_val, /**< function object */
   return ret_value;
 } /* test262_eval_script */
 
+/**
+ * $262.evalScript
+ *
+ * A function which accepts a string value as its first argument and executes it
+ *
+ * @return completion of the script parsing and execution.
+ */
+static jerry_value_t
+test262_create_realm (const jerry_value_t func_obj_val, /**< function object */
+                      const jerry_value_t this_p, /**< this arg */
+                      const jerry_value_t args_p[], /**< function arguments */
+                      const jerry_length_t args_cnt) /**< number of function arguments */
+{
+  (void) func_obj_val; /* unused */
+  (void) this_p; /* unused */
+  (void) args_p; /* unused */
+  (void) args_cnt; /* unused */
+
+  jerry_value_t result_object = jerry_create_object ();
+
+  jerry_value_t name_val = jerry_create_string ((const jerry_char_t *) "global");
+  jerry_value_t realm_val = jerry_create_realm ();
+
+  jerry_value_t result_val = jerry_set_property (result_object, name_val, realm_val);
+
+  jerry_release_value (result_val);
+  jerry_release_value (realm_val);
+  jerry_release_value (name_val);
+
+  return result_object;
```

Comment:
I think this is incorrect.

According the int `INTERPRETING.md` in test262
```
createRealm - a function which creates a new ECMAScript Realm, defines this API on the new realm's global object, and returns the $262 property of the new realm's global object
```

So what I think we need to do there:
1, Replace `static void register_test262 (void)` to `static jerry_value_t register_test262 (jerry_value_t global_obj)`.
 - which returns the registered `test262` object to the given `global_obj`.
 - Releases the given `global_obj`

2, Replace the current usage of `register_test262()` to `jerry_release_value(register_test262 (jerry_get_global_object ()))`
3, In `test262_create_realm` create a new realm `realm_val` and return with `register_test262 (new_realm)`;

Reply:
Good idea, it seems it fixed another 4 tests.